src/showcase/ui-group/ButtonCards.tsx

```diff
@@ -71,6 +72,12 @@ const data = [
     link: '/components/button/shine-button',
     component: <ShineButton />,
   },
+  {
+    id: 11,
+    title: 'Text Reveal',
+    link: '/docs/components/button/text-reveal-button',
```

Comment:
Also, this link will work because I set up middleware to redirect `docs/` to active sites, but could you change this to `/components/button/text-reveal-button` as it is the true link that it will be redirected to?